cpp/src/io/parquet/compact_protocol_reader.cpp

```diff
@@ -16,6 +16,9 @@
 
 #include "compact_protocol_reader.hpp"
 
+#include "parquet.hpp"
```

Comment:
Please remove the empty line above :smile:

Reply:
It's weird. Definitely clang-format bug :sweat:

Reply:
Yeah. Maybe a special group for header with the same root as the compilation unit 🤷

Reply:
@harrism Would you mind looking into this please?